src/dscom.build/dSPACE.Runtime.InteropServices.BuildTasks.Tools.targets

```diff
@@ -49,6 +49,7 @@
       <_DsComArguments>tlbexport</_DsComArguments>
       <_DsComArguments Condition="'$(DsComTypeLibraryUniqueId)' != '' AND '$(DsComTypeLibraryUniqueId)' != '$([System.Guid]::Empty.ToString())'">$(_DsComArguments) --overridetlbid "$(DsComTypeLibraryUniqueId)"</_DsComArguments>
       <_DsComArguments Condition="'$(DsComToolVerbose)' == 'true'"> $(_DsComArguments) --verbose</_DsComArguments>
+      <_DsComArguments> $(_DsComArguments) --tlbembed</_DsComArguments>

```

Comment:
Don't currently know what is going wrong here but during the acceptance tests we get 
```
dscom.exe      tlbexport --tlbembed "D:\a\dscom\dscom\examples\32bit\comtestdotnet\bin\x64\Release\net48\comtestdotnet.dll"  --out "D:\a\dscom\dscom\examples\32bit\comtestdotnet\bin\x64\Release\net48\\comtestdotnet.tlb" 
```
while it should be 
````
dscom.exe      tlbexport "D:\a\dscom\dscom\examples\32bit\comtestdotnet\bin\x64\Release\net48\comtestdotnet.dll"  --tlbembed  --out "D:\a\dscom\dscom\examples\32bit\comtestdotnet\bin\x64\Release\net48\\comtestdotnet.tlb" `
```
because the pattern should be `dscom tlbexport <Assembly> [options]`

Reply:
@SOsterbrink Not really. As tlbembed is realized as a switch, it should be like --verbose just to be enough.

Reply:
I fixed so that the ordering of the arguments is a bit more sensible even though it's not strictly necessary for reasons Carsten pointed out.